### PR TITLE
add depth anything notebook

### PR DESCRIPTION
**Pull request by eaidova**
Ticket: CVS-130636

### CODE REVIEW

File: notebooks/281-depth-anything/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+ Monodepth Demo
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F281-depth-anythingh%2F281-depth-anything.ipynb)
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/281-depth-anything/281-depth-anything.ipynb)
+
+![depth_map.png](https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/a9a16658-512f-470c-a33c-0e1f9d0ae72c)
+
+[Depth Anything](https://depth-anything.github.io/) is a highly practical solution for robust monocular depth estimation. Without pursuing novel technical modules, this project aims to build a simple yet powerful foundation model dealing with any images under any circumstances.
+The framework of Depth Anything is shown below. it adopts a standard pipeline to unleashing the power of large-scale unlabeled images. 
+![image.png](attachment:df8f001d-8132-4ea5-bec2-b6b9f55089a1.png)
```
Comment (itrushkin) [inline notebooks/281-depth-anything/README.md:10]: Image is not displayed

File: notebooks/281-depth-anything/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+ Monodepth Demo
```
Comment (itrushkin) [inline notebooks/281-depth-anything/README.md:1]:
```suggestion
# Monodepth Demo

**Review by itrushkin — CHANGES_REQUESTED** [review]

File: notebooks/281-depth-anything/README.md (hunk 2 above)
Comment (eaidova) [reply notebooks/281-depth-anything/README.md]: it is accidentally copied from another notebook, title changed

File: notebooks/281-depth-anything/README.md (hunk 1 above)
Comment (eaidova) [reply notebooks/281-depth-anything/README.md]: updated

File: notebooks/281-depth-anything/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+![image.png](https://depth-anything.github.io/static/images/pipeline.png)
+
+More details about model can be found in [project web page](https://depth-anything.github.io/), [paper](https://arxiv.org/abs/2401.10891), and official [repository](https://github.com/LiheYoung/Depth-Anything)
+
+In this tutorial we will explore how to convert and run DepthAnything using OpenVINO.
+
+## Notebook Contents
+
+This notebook demonstrates Monocular Depth Estimation with the [DepthAnything](https://github.com/LiheYoung/Depth-Anything) in OpenVINO.
+
```
Comment (itrushkin) [inline notebooks/281-depth-anything/README.md:19]: Please add a list of tutorial steps to this section

File: notebooks/281-depth-anything/281-depth-anything.ipynb [hunk 4]
```diff
@@ -59,6 +72,8 @@
    "id": "84f1c16c-0d0f-49aa-bc45-c1fb7a59f0bf",
    "metadata": {},
    "source": [
+    "## Load and run PyTorch model\n",
+    "[back to top ⬆️](#Table-of-contents:)\n",
     "## Load and run PyTorch model"
```
Comment (itrushkin) [inline notebooks/281-depth-anything/281-depth-anything.ipynb:77]: Some markdown headers are duplicated. We should check our TOC generation script

File: notebooks/281-depth-anything/README.md (hunk 3 above)
Comment (eaidova) [reply notebooks/281-depth-anything/README.md:19]: done

**Comment by eaidova** [comment]
@itrushkin please take a look one more time, repeated headers are also removed

**Review by itrushkin — APPROVED** [review]

**Pull request merged**